README.md

```diff
@@ -2539,6 +2539,37 @@ no parameters.
   end
   ```
 
+* <a name="accessor_mutator_method_names"></a>
+  For more complex accessors and mutators, avoid prefixing methods with
```

Comment:
remove "more complex" here. I understand what you mean, but I believe this wording is more confusing than helpful.
methods -> method names